packages/dai-plugin-migrations/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@makerdao/dai-plugin-migrations",
+  "description": "Plugin to add migrations to dai.js",
+  "version": "0.1.0",
+  "license": "MIT",
+  "main": "dist",
+  "scripts": {
+    "build": "yarn prepublishOnly",
+    "prepublishOnly": "./scripts/prepublish.sh",
+    "testchain": "../../scripts/run-testchain.sh --ci",
```

Comment:
note that in the other packages i've moved the `--ci` argument to the `yarn test` command, so that i can run `yarn testchain` without arguments to create a long-running testchain process; you may want to do the same here for consistency's sake